web/streetmanager/srwr_process.php

```diff
@@ -0,0 +1,345 @@
+<?php
+
+require_once __DIR__ . '/../../conf/config';
+require_once __DIR__ . '/../fns.php';
+require __DIR__ . '/../../vendor/autoload.php';
+
+$path = './srwr_data/';
+$printrow = [];
+$companies = [];
+
+// Delete any previously downloaded files
+deleteFiles();
```

Comment:
I would probably split this script into two scripts - one to clean/fetch/unzip the data. And one to process it. That former one could even then be a bash script rather than PHP, if all it's doing is `rm`, `curl` and `unzip`, if there's nothing programmatic to do

Reply:
I can do...I would guess as the php is done, we could just split that out though?